src/ansys/units/cfg.yaml

```diff
@@ -5,16 +5,16 @@
 # Dimension type : Order
 
 dimension_order:
-  Mass: 1
-  Length: 2
-  Time: 3
-  Temperature: 4
-  Temperature Difference: 4
-  Angle: 5
-  Chemical Amount: 6
-  Light: 7
-  Current: 8
-  Solid Angle: 9
+  MASS: 0
```

Comment:
@ansaboutin awesome to see this change as it brings about consistency and simplicity. I also believe that we can altogether elminate it from here as it is not genuinely configurable: it is defioned in the code.

I'm a bit confused as all my comments of yesterday seem to have disappeared from the PR, but you have seen them as they have been acted on.

Reply:
Yes, when comments are resolved they disappear.  You can still go back and see them if you go to this and select it:

![image](https://github.com/ansys/pyansys-units/assets/18702052/5e129d76-4a8e-41f0-882c-80ba9ea6ba41)